src/chatlist/chatList.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { withStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import styles from "./styles";
+import Divider from "@material-ui/core/Divider";
+import Button from "@material-ui/core/Button";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import NotificationImportant from "@material-ui/icons/NotificationImportant";
+
+class ChatListComponent extends React.Component {
+  newChat = () => {
+    console.log("New chat click");
+  };
+
+  selectChat = (index) => {
+    console.log("select chat", index);
+  };
+  render() {
+    const { classes } = this.props;
+
+    if (this.props.chats.length > 0) {
+      return (
+        <main className={classes.root}>
+          <Button
+            variant="contained"
+            fullWidth
+            color="primary"
+            className={classes.newChatBtn}
+            onClick={this.newChat}
+          >
+            New Message
+          </Button>
+          <List>
+            {this.props.chats.map((_chat, _index) => {
```

Comment:
I'm curious about your function prop variable names here starting with the _underscore.
I've only seen underscored variable names used in classes, when the author intends for the variable to be "private" and unreachable outside of the class. 

It's certainly not wrong but I just wanted to mention it's not very common.

Reply:
It's written like this in the tutorial, that's why I have hesitated to delete them. I thought there must be some reason to use the underscore in such cases. If there is not, I will remove them.